Sources/Workspace/Workspace.swift

```diff
@@ -251,6 +254,12 @@ public class Workspace {
     /// The manifest loader to use.
     let manifestLoader: ManifestLoaderProtocol
 
+    /// The tools version currently in use.
+    let currentToolsVersion: ToolsVersion
```

Comment:
This is for testing purposes (dependency injection). But it is a default parameter in init() so it is not expected to be passed by clients.